dspace-api/src/main/java/org/dspace/handle/HandleServiceImpl.java

```diff
@@ -420,4 +420,18 @@ public String parseHandle(String identifier) {
 
         return null;
     }
+
+    /**
+     * Returns a list of handles of items that have been deleted.
+     *
+     * @param context
+     *            DSpace context
+     * @return The handle for object, or null if the object has no handle.
+     * @exception SQLException
+     *                If a database error occurs
+     */
```

Comment:
minor things, this should go in the interface where the other public methods are documented